src/components/ProjectsContainer.js

```diff
@@ -19,6 +19,10 @@ function ProjectsContainer({styles, projects, setHeroProject}) {
     }
   };
 
+  const handleHeroProject = (project) => {
```

Comment:
Why is this needed? it's a single method that calls another single method.

Reply:
There are two reasons, both no longer make sense in the way it's set up now. It's a bit superfluous. The plan was to handle the scrollTo in the function, but adding an anchor tag proved easier/semantic. The other was I had heard that having inline functions on an onClick can add extra rerenders, but I am still doing that so I might as well, not have an inline function that calls another function when I can just call setHeroProject.